### PR TITLE
Issue 46: [ENG] Add Dark / Light Mode to BeatMarker

### PR DESCRIPTION
**Pull request by HarisRen**
#### What does this PR do?
1. Add Light / Dark mode 
2. Some css corrections

#### What are the relevant issues?
[#46](https://app.zenhub.com/workspaces/growth---workboard-5fa64905617160001644131b/issues/accusonus/beatmarker-private/46)

### CODE REVIEW

**Comment by ccristos13** [comment]
@SpyrosSiannasAccusonus is there any chance to review this asap?

**Comment by SpyrosSiannasAccusonus** [comment]
Heey @ccristos13. I think I'll be able to do it today, yeah!

File: src/css/style.css
Comment (thanos-n) [inline src/css/style.css]: The wave color in the light theme makes it a bit hard to distinguish from the color of the available (but not selected) markers (especially for longer audio tracks). See screenshot below:

Screenshot 2021-11-16 at 4 35 49 PM



It would be nice to change either this one or the color of the markers (its set [here](https://github.com/accusonus/beatmarker-gpl/blob/ce121593b2ee6603c6021630f3e5dae1fcb05899/src/js/index.js#L701) currently - also, this could be set to get the value from the theme instead) so that there's enough contrast between the markers and the waveform in light mode.

File: src/js/index.js
Comment (thanos-n) [inline src/js/index.js:902]: Because this is called only when the waveform is initialized, the color of the bars does not get updated when switching themes. We could perhaps target the `.wavesurfer-handle` class inside `themeInit()` and set the backgroundColor property as `!important` to override the element styling.

**Review by thanos-n — CHANGES_REQUESTED** [review]
Changes look good overall! ☀️ 🌙  I left two comments to be addressed before merging!

File: src/css/style.css
Comment (accubill) [reply src/css/style.css]: Hi guys,

Thano, could you please share a screenshot of how you propose we handle this?

Thank you!

Comment (thanos-n) [reply src/css/style.css]: > Hi guys,
> 
> Thano, could you please share a screenshot of how you propose we handle this?
> 
> Thank you!

Hey @accubill ! Something like this looks more legible I think. I changed both the waveform and the color of the unselected markers for the light theme. What do you think?

Screenshot 2021-11-17 at 10 11 59 AM

- waveform color : #434343
- unselected/available markers: #C7C7C7

Comment (accubill) [reply src/css/style.css]: What am I to think other than: that's both legible and **LEGIT**!

Comment (HarisRen) [reply src/css/style.css]: Hello guys. Thanks for the great suggestions. The light and dark is like this: 
image

image

**Review by thanos-n — APPROVED** [review]
Changes look good! Approving so that we can merge fast.

There is one more detail regarding the light theme. From what I've seen the `.dropzone` is not using the `--ui-dim-color-light` and `--text-color-light` when we're on the light theme. @HarisRen if you find some time and think this can be easily fixed, ping me. Otherwise feel free to merge!

**Comment by HarisRen** [comment]
I added the classes for light color in dropzone. Please check and if ti's ok proceed. Thanks for the comment

**Comment by HarisRen** [comment]
@thanos-n Checked. Great job. Thanks. Shall we merge?

**Comment by thanos-n** [comment]
> @thanos-n Checked. Great job. Thanks. Shall we merge?

Yes, merged!

**Review by SpyrosSiannasAccusonus — APPROVED** [review]
Checked too. I think we're good.

**Comment by VasilisBachras** [comment]
Checked too from the growth-team's side, all good.

**Pull request merged**
